src/main/webapp/app/shared/components/open-repository-button/open-repository-button.component.html

```diff
@@ -0,0 +1,45 @@
+<ng-container>
+    @if (participations?.length && participations?.length === 1) {
+        <button
```

Comment:
if this button is navigating to a new page, please use an `a` HTML tag instead of a `button` tag